src/Illuminate/Mail/composer.json

```diff
@@ -21,7 +21,7 @@
         "illuminate/support": "5.6.*",
         "psr/log": "~1.0",
         "swiftmailer/swiftmailer": "~6.0",
-        "tijsverkoyen/css-to-inline-styles": "~2.2"
+        "tijsverkoyen/css-to-inline-styles": "dev-master"
```

Comment:
And here.

Reply:
@GrahamCampbell This has been tagged 27 days ago (2.2.1) is it worth changing it from dev-master. Seems that laravel shouldn't really rely on a branch as a dependency.

Reply:
Yes, this was updated 27 days ago.